core/templates/dev/head/pages/splash/splash.html

```diff
@@ -121,22 +121,22 @@ <h1 class="oppia-splash-h1" translate="I18N_SPLASH_TITLE"></h1>
         </div>
 
         <div class="oppia-splash-button-container">
-          {% if not user_is_logged_in %}
             <!-- User must complete the registration process. -->
             <button type="button"
+                    ng-if="!userIsLoggedIn"
                     ng-click="onRedirectToLogin('/creator_dashboard?mode=create')"
                     class="btn oppia-splash-button oppia-splash-button-create oppia-transition-200"
                     style="left: -webkit-calc(130px + 15%); left: -moz-calc(130px + 15%); left: -o-calc(130px + 15%); left: calc(130px + 15%); bottom: 210px; z-index: 10;"
                     translate="I18N_ACTION_CREATE_LESSON">
             </button>
-          {% else %}
+
```

Comment:
I guess we don't any extra line between two button elements, we can just remove it.

Reply:
Done! (in next commit) :)